src/components/Header/Header.tsx

```diff
@@ -44,6 +44,21 @@ export const Header = ({ breadcrumbsProps }: { breadcrumbsProps?: Breadcrumbspro
   const { pathname } = useLocation();
   const noBreadcrumb = !['/', '/allservices', '/favoritedservices'].includes(pathname);
 
+  const useWindowWidth = () => {
```

Comment:
Can we have a separate file for this hook? It might be useful in other places later on. There is a hook directory with some files already.